### PR TITLE
fix pool + add github run tox

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

**Comment by Arstory-MK** [comment]
Thank you for taking your time to fix the problem!

File: hcga/features/centralities_basic.py [hunk 1]
```diff
@@ -134,7 +134,7 @@ class CentralitiesBasic(FeatureClass):
            Internet Mathematics 10.3-4 (2014): 222-262.
     """
 
-    modes = ["fast", "medium", "slow"]
+    modes = ["medium", "slow"]
```
Comment (arnaudon) [inline hcga/features/centralities_basic.py:137]:
```suggestion
    modes = ["fast", "medium", "slow"]

```

File: hcga/features/node_features.py [hunk 2]
```diff
@@ -91,7 +91,7 @@ class NodeFeatures(FeatureClass):
 
     """
 
-    modes = ["fast", "medium", "slow"]
+    modes = ["medium", "slow"]
```
Comment (arnaudon) [inline hcga/features/node_features.py:94]:
```suggestion
    modes = ["fast", "medium", "slow"]

```

File: hcga/features/shortest_paths.py [hunk 3]
```diff
@@ -78,7 +78,7 @@ class ShortestPaths(FeatureClass):
 
     """
 
-    modes = ["fast", "medium", "slow"]
+    modes = ["medium", "slow"]
```
Comment (arnaudon) [inline hcga/features/shortest_paths.py:81]:
```suggestion
    modes = ["fast", "medium", "slow"]

```

File: hcga/features/shortest_paths.py [hunk 4]
```diff
@@ -79,6 +79,7 @@ class ShortestPaths(FeatureClass):
     """
 
     modes = ["fast", "medium", "slow"]
+
```
Comment (arnaudon) [inline hcga/features/shortest_paths.py:82]:
```suggestion
```

File: hcga/features/node_features.py [hunk 5]
```diff
@@ -92,6 +92,7 @@ class NodeFeatures(FeatureClass):
     """
 
     modes = ["fast", "medium", "slow"]
+
```
Comment (arnaudon) [inline hcga/features/node_features.py:95]:
```suggestion
```

File: hcga/features/centralities_basic.py [hunk 6]
```diff
@@ -135,6 +135,7 @@ class CentralitiesBasic(FeatureClass):
     """
 
     modes = ["fast", "medium", "slow"]
+
```
Comment (arnaudon) [inline hcga/features/centralities_basic.py:138]:
```suggestion
```

**Pull request merged**